js/flightlog_parser.js

```diff
@@ -487,7 +546,7 @@ var FlightLogParser = function(logData) {
                 } else {
                     that.sysConfig[fieldName] = parseInt(fieldValue, 10);
                 }
-                break;
```

Comment:
Are you talking about the `break;`? The indentation is correct. This is the removed part and has been left as the others.